dnf/base.py

```diff
@@ -926,7 +926,9 @@ def add_remote_rpm(self, path):
             # download remote rpm to a tempfile
             path = dnf.util.urlopen(path, suffix='.rpm', delete=False).name
             self._add_tempfiles([path])
-        return self.sack.add_cmdline_package(path)
+        pkg_rpm = self.sack.add_cmdline_package(path)
+        self._setup_excludes_includes()
+        return pkg_rpm
 
```

Comment:
This is not a right solution e.g. it won't work for packages not presented in any enabled repository. Focus more on `_setup_excludes_includes()`.